.travis.yml

```diff
@@ -26,6 +32,7 @@ before_install:
   - travis_retry composer self-update && composer --version
 
 install:
+  - if [ "$SYMFONY_VERSION" != "" ]; then travis_retry composer require "symfony/symfony:${SYMFONY_VERSION}" --prefer-source --no-update -n; fi;
```

Comment:
From the bundle's composer.json:

```json
    "require": {
        "symfony/framework-bundle": "^2.7.3",
        "symfony/dependency-injection": "^2.7.3",
        "puli/composer-plugin": "^1.0-beta8",
        "puli/symfony-bridge": "^1.0-beta3"
    },
    "require-dev": {
        "symfony/twig-bundle": "^2.7.3",
        "puli/twig-extension": "^1.0-beta7",
        "puli/discovery": "^1.0-beta8",
        "puli/cli": "^1.0-beta8",
        "phpunit/phpunit": "^4.6",
        "sebastian/version": "^1.0.1"
    },
```

Hence this line should be:

```
  - if [ "$SYMFONY_VERSION" != "" ]; then travis_retry composer require "symfony/framework-bundle:${SYMFONY_VERSION}" "symfony/dependency-injection:${SYMFONY_VERSION}" --no-update -n; travis_retry composer require "symfony/twig-bundle:${SYMFONY_VERSION}" --no-update -n; fi;
```

As far as I understand the --prefer-source can be removed since we're doing --no-update anyway.

@pkruithof could you update this PR?